components/dot_menu/dot_menu.scss

```diff
@@ -0,0 +1,15 @@
+.dotmenu-dropdown-menu {
+    .Menu__content {
+        left: inherit;
```

Comment:
I don't think we need this here. This might even be prone to bugs when some parent element has a value for `left` defined